taxi/views.py

```diff
@@ -61,17 +62,25 @@ class CarListView(LoginRequiredMixin, generic.ListView):
 class CarDetailView(LoginRequiredMixin, generic.DetailView):
     model = Car
 
+    def post(self, request, **kwargs):
```

Comment:
it would be better to write separate classview for this function

Reply:
I don't know how to properly do this functionality using Classbase so I reworked this logic using Functionbase